changelogs/fragments/1449-zypper_repository-change_custom_repository_integration_test.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - zypper_repository - change custom repository integration test to fix upstream issue (https://github.com/ansible-collections/community.general/issues/1446).
```

Comment:
Test-only changes do not need a changelog.

Reply:
Thanks, I didn't know this. I've removed the commit which adds the changelog. 
https://github.com/ansible-collections/community.general/compare/cee77ca11a4b8dc09d26ee2a8c8e0086b4a80fed..83961e0f268c2965a61caff496bc71ac998597e3